src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java

```diff
@@ -144,24 +145,30 @@ private ReleaseCreationInformation searchEventInFollowingBlocks(
             blockNumber,
             block.getTransactionsList().size()
         );
-        for (Transaction transaction : block.getTransactionsList()) {
-            TransactionInfo transactionInfo = receiptStore.getInMainChain(transaction.getHash().getBytes(), blockStore).orElse(null);
-            TransactionReceipt transactionReceipt = transactionInfo.getReceipt();
-            transactionReceipt.setTransaction(transaction);
-            Optional<ReleaseCreationInformation> optionalReleaseCreationInformation =
-                getInformationFromEvent(block, transactionReceipt, btcTransaction, rskTxHash, informingRskTxHash);
-            if (optionalReleaseCreationInformation.isPresent()) {
-                return optionalReleaseCreationInformation.get();
+        for (Transaction tx : block.getTransactionsList()) {
```

Comment:
```suggestion
        for (Transaction rskTx: block.getTransactionsList()) {
```

I think it is necessary to use the prefix `rsk`(or `btc` accordingly) since we are dealing with both transactions. Also, to be consistent with this naming convention we've been using so far.

Reply:
I see, just updated the name of the var.